includes/polling/os/elg-ipecs-es.inc.php

```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * ericsson-es.inc.php
+ * ericsson-ucp.inc.php
```

Comment:
This and the line two below need updating to reference the new file name

Reply:
is [this](https://github.com/librenms/librenms/pull/7289/commits/e4dbdb73e9bb548fe5d1e11c4dbcf86fecea6a90) right?